bolero-generator/src/arbitrary.rs

```diff
@@ -0,0 +1,70 @@
+use core::{cmp, marker::PhantomData};
+
+use arbitrary::{Arbitrary, Unstructured};
+
+use crate::{gen_with, Driver, ValueGenerator};
```

Comment:
nit:

```suggestion
use arbitrary::{Arbitrary, Unstructured};
use core::{cmp, marker::PhantomData};
use crate::{gen_with, Driver, ValueGenerator};
```

Reply:
FWIW, this is how `rust-analyzer` and `rustfmt` auto-formatted my imports after I just used vscode's "auto-import missing stuff" feature — not like that's important, I've handled it :)